app/pages/docs/why-blitz.mdx

```diff
@@ -1,73 +1,75 @@
 ---
-title: Why use Blitz instead of Next.js?
-sidebar_label: Why Blitz instead of Next.js?
+title: Waarom Blitz gebruiken in plaats van Next.js
+sidebar_label: Waarom Blitz gebruiken in plaats van Next.js
 ---
 
-### 1. Fullstack instead of Frontend
+### 1. Fullstack in plaats van frontend
 
-Next.js is mostly a frontend framework. It's designed for building a frontend that connects to some other API. Next.js isn't set up by default for building a true fullstack application.
+Next.js is vooral een frontend framework. Het is ontworpen voor het bouwen van een frontend die verbinding maakt met een andere API. Next.js is niet standaard ingesteld om een echte fullstack applicatie te bouwen.
 
-Blitz adds all the missing features and functionality that turns Next into a true fullstack framework. These key features include direct database access, middleware, and authentication.
+Blitz voegt alle ontbrekende functies en functionaliteit toe die van Next een echt fullstack framework maken. Deze belangrijke functies omvatten directe toegang tot de database, middleware, en authenticatie.
 
-### 2. Data Layer
+### 2. Data layer
 
-The new Blitz data layer is a novel technology for eliminating the API while retaining client-side rendering. In a Blitz app, you can import your server code directly into the frontend so you don't need to build an API and do data fetching from the frontend. At build time, Blitz automatically inserts an API call that runs the server code on the server. Essentially, Blitz abstracts your API into a compile step.
+De Blitz data layer is een nieuwe technologie voor het elimineren van de API met behoud van client-side rendering. In een Blitz app kun je je servercode rechtstreeks in de frontend importeren, zodat je geen API hoeft te bouwen en geen data hoeft op te halen van de frontend. Bij het bouwen voegt Blitz automatisch een API-call toe die de servercode op de server uitvoert. Eigenlijk abstraheert Blitz je API in een compileerstap.
 
-This is a game changer for React app development because it eliminates an entire portion of traditional React app architecture. This means it's easier to learn, faster to develop, and more fun to build things!
+Dit is een doorbraak in de ontwikkeling van React-apps, omdat het een heel deel van de traditionele React-architectuur elimineert. Dit betekent dat het makkelijker te leren, sneller te ontwikkelen, en leuker om dingen te bouwen is!
```

Comment:
Ik denk dat er een verschil is tussen "React-architectuur" (de architectuur van React) of "React app-architectuur" (de architectuur van een React app).